release_notes/45-release-notes.adoc

```diff
@@ -0,0 +1,418 @@
+
+[id="build-in-email-notifier-acscs_{context}"]
+=== Built-in email notifier in {product-title-managed-short}
+
+If you are using {rh-rhacscs-first}, you can use the new built-in email notifier to send email notifications without configuring a third-party email provider.
+
+[id="scanner-v4-ga_{context}"]
+=== Scanner V4 is generally available
+
+Scanner V4 is now generally available. It integrates the features of the existing StackRox Scanner and the upstream Clair V4 Scanner and provides the following improvements:
+
+* *Consistent and accurate scanning*: Scanner V4 provides reliable vulnerability scan results across the entire Red{nbsp}Hat product ecosystem.
+* *Expanded language and operating system support* :
+** {product-title-short} now supports Golang for language vulnerability scanning.
+** {product-title-short} now supports Oracle Linux, SUSE Linux Enterprise, and Photon OS for operating system scanning.
+* *Comprehensive vulnerability database source*: Scanner V4 uses link:https://osv.dev/[OSV.dev] as the vulnerability database source for all supported programming language packages.
+
+For more details, see xref:../operating/examine-images-for-vulnerabilities.adoc#about-scanner-v4_examine-images-for-vulnerabilities[About {product-title-short} Scanner V4].
+
+[id="roxctl-installation-github-action_{context}"]
+=== `roxctl` installation GitHub action
+
+The link:https://github.com/stackrox/roxctl-installer-action[roxctl-installer-action] GitHub action is now available. You can use it to install `roxctl` in your GitHub workflows and run `roxctl image check` and `roxctl image scan` commands in your CI pipelines.
+
+[id="bring-your-own-pki_{context}"]
+=== Bring your own PKI for signature verification
+
+Before this release, you could only verify image signatures against pre-configured keys. With {product-title-short} 4.5, Red{nbsp}Hat has extended the image signature verification to include verifying certificates. This method is available in addition to verifying public keys, allowing you to bring your own public key infrastructure (PKI) to verify signatures.
+
+[id="build-time-network-policy-tools-updates_{context}"]
+=== Build-time network policy tools updates
+
+This release introduces an update to the `roxctl netpol generate` command.
+
+* By default, the command uses port `53` for DNS connections.
+* You can use the `--dnsport` option to override the default DNS port. If you are using {ocp}, you must always change the port when generating network policies with `roxctl` because {ocp} uses port `5353`, not port `53`. For example, `roxctl netpol generate --dnsport 5353 <other-options>`.
+
+To simplify usage for {ocp} and other systems that use named ports, in a future release, Red{nbsp}Hat plans to enhance the `--dnsport` option to accept strings, such as `dns`, in addition to numbers. With this change, you will no longer need to remember specific port numbers.
```

Comment:
```suggestion
To simplify usage for {ocp} and other systems that use named ports,  Red{nbsp}Hat plans to enhance the `--dnsport` option in a future release to accept strings in addition to numbers. With that change, the generated network policy can use a named port (such as `dns`)  instead of a specific port number, which is a more portable approach.
```

Reply:
It's not about remembering port numbers, kindly accept the proposed change

Reply:
I inferred it from https://github.com/openshift/openshift-docs/pull/78414#discussion_r1678253468

> ... this would eliminate the need for users to know the specific port number. 

Thanks, I'll update.